src/ftsh.c

```diff
@@ -69,18 +42,53 @@ void ftsh_loop(void)
     do {
         input = readline("ftsh> ");
         args = ftsh_get_input(input);
-        status = ftsh_execute(args);
+        status = ftsh_execute(args, pf);
 
         free(input);
         free(args);
     } while (status);
 }
 
 
+char* get_path(int argc, char **argv) {
+    if (argc <= 1) {
+        return NULL;
+    }
+
+    return (argv[1] != NULL) ? argv[1] : NULL;
+}
+
 int main(int argc, char **argv)
 {
 
-    ftsh_loop();
+    // Config
+    char* path = get_path(argc, argv);
+    if (argc == 1 || path == NULL) {
+    	printf("Usage: ./ftsh <text_search_file>\n");
+    	exit(0);
+    }
+    FILE *pf = fopen(path, "r");
+    int mode = 1;
+
+    if (pf == NULL) {
+        perror("File could not be opened");
+        exit(1);
+    }
+
+    // Interactive mode
+    if (mode == 1) {
+        printf("...starting interactive mode...\n");
```

Comment:
`printf`s like this should be omitted or relegated to the `DEBUG` level if y'all want to use a logging library.

Reply:
Commented out for now. I will let @mbronckers make the decision on how to proceed bc he wrote the code for this part.

Reply:
Deleted printf